core/domain/topic_services.py

```diff
@@ -945,6 +961,29 @@ def check_can_edit_subtopic_page(user):
     return False
 
 
+def deassign_user_from_all_topics(committer, user_id):
+    """Deassigns given user from all topics assigned to them.
+
+    Args:
+        committer: UserActionsInfo. UserActionsInfo object for the user
+            who is performing the action.
+        user_id: str. The ID of the user.
+
+    Raises:
+        Exception. The committer does not have rights to modify a role.
+    """
+    topic_rights_list = get_topic_rights_with_user(user_id)
+    for topic_rights in topic_rights_list:
+        topic_rights.manager_ids.remove(user_id)
+        commit_cmds = [topic_domain.TopicRightsChange({
+            'cmd': topic_domain.CMD_REMOVED_MANAGER_ROLE,
+            'assignee_id': user_id
```

Comment:
Maybe should be removed_user_id rather than assignee_id. The latter suggests that the person is being assigned a role, which is not the case here.

Reply:
Done